### ISSUE TITLE
architecture is "x86' on arm - kestrel will try loading x86 libuv.dll

### ISSUE THREAD

**Issue by moozzyk** [aspnet/KestrelHttpServer#106]
https://github.com/aspnet/KestrelHttpServer/blob/dev/src/Microsoft.AspNet.Server.Kestrel/KestrelEngine.cs#L36

Related: 

https://github.com/aspnet/dnx/issues/2063

**Comment by muratg** [comment]
@DamianEdwards is supporting Kestrel on ARM is out of scope for V1?

Also please see: https://github.com/aspnet/dnx/issues/2063

**Comment by moozzyk** [comment]
Actually this should resolve itself semi-automatically when dnx starts loading native dlls automatically. - then we will be able to remove the code where we have the check.

**Comment by Tragetaschen** [comment]
Uhm, current Kestrel works fine on Linux on ARM given a proper libuv. Making it run on Windows on ARM should be really straight forward even now.

**Comment by matizk144** [comment]
> Uhm, current Kestrel works fine on Linux on ARM given a proper libuv. Making it run on Windows on ARM should be really straight forward even now

Could you check latest version on Linux arm? Unfortunatelly in my RP2 doesn't work (Windows IOT)

**Comment by moozzyk** [comment]
This was actually fixed in d4853f9b7 where we started using the built-in support for loading native libraries instead of custom code we had before where we had this check.
@matizk144 - I believe what you are seeing is a different issue. You did not publish for win10 arm but used the default runtime id for which libuv is not defined. This is bug #340.
I am closing this bug and will follow up on this in the other issue - I did hit the same thing when I first tried Kestrel on R-Pi2.
